perun-core/src/main/java/cz/metacentrum/perun/core/implApi/AttributesManagerImplApi.java

```diff
@@ -159,6 +159,31 @@
 	List<Attribute> getAttributes(PerunSession sess, Resource resource, Member member) throws InternalErrorException;
 
 	/**
+	 * Get all <b>non-empty</b> attributes associated with the member in the group.
```

Comment:
I know, this is copy paste, so the right mistake is in javadoc at all, but please, say there that "Get all non-empty, not-virtual ... " cause only core,opt,def are get there.